patch.cabal

```diff
@@ -40,9 +35,13 @@ library
                , dependent-sum >= 0.6 && < 0.8
                , lens >= 4.7 && < 5
                , semigroupoids >= 4.0 && < 6
+               , these >= 0.4 && < 1.2
                , transformers >= 0.5.6.0 && < 0.6
                , witherable >= 0.3 && < 0.4
 
+  if impl(ghc < 8.6) -- really, if base < 8.12
+    build-depends: base-orphans >= 0.8 && < 0.9
```

Comment:
We could get rid of this, and just not provide `:.:` and `:*:` `Additive` instances before GHC 8.6. I don't care which.